requirements/edx/base.in

```diff
@@ -146,7 +146,6 @@ requests-oauthlib                   # Simplifies use of OAuth via the requests l
 random2
 rules                               # Django extension for rules-based authorization checks
 simplejson
-sailthru-client                     # For Sailthru integration
```

Comment:
This doesn't actually change any of the generated requirements `txt` files because `edx-ace` still pulls it in.